src/Views/Pokes/Poke.js

```diff
@@ -0,0 +1,98 @@
+import { useHistory } from "react-router-dom";
+
+import BookListItem from "../../SharedComponents/BookListItem";
+
+import "./Pokes.scss";
+
+const PokeReceived = ({ bookList, creationDate, sender, read, handleOnClick }) => {
+  return (
+    <div className={!read && "newPoke"}>
+      {bookList.length > 0 ? (
+        <>
+          <p>
+            On {creationDate} <b>{sender.name}</b> showed interest in the
+            following books in your collection:
+          </p>
+          <ul>{bookList}</ul>
+          <p>and invited you to browse his/hers collection.</p>{" "}
+        </>
+      ) : (
+        <p>
+          On {creationDate} <b>{sender.name}</b> showed interest in your
+          collection and invited you to browse his/hers collection
+        </p>
+      )}
+      <button onClick={handleOnClick}>
+        Visit <b>{sender.name}</b>'s profile
+      </button>
+    </div>
+  );
+};
+
+const PokeSent = ({
+  bookList,
+  creationDate,
+  recipient,
+  handleOnClick,
+}) => {
+  return (
+    <div>
+      {bookList.length > 0 ? (
+        <>
+          <p>
+            On {creationDate} you showed interest in the following books in{" "}
```

Comment:
`{" "}` - what is it for? :P

Reply:
It's Prettier and I guess it's important, because when I delete it there is no space between words ;)